sdk/storage/storage-file-datalake/src/utils/PathClientUtils.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { Path } from "../generated/src/operations";
+import { Pipeline } from "../Pipeline";
+import { StorageClient } from "../StorageClient";
+
+/**
+ * A UtilsPathClient represents a URL to the Azure Storage path (directory or file) to
+ * help to construct a path client to expose Path context with blob endpoint.
+ */
+export class UtilsPathClient extends StorageClient {
```

Comment:
Either rename to BlobPathClient
Or extend PathClient

Reply:
Rename to PathClientInternal